internal/users/create.go

```diff
@@ -65,7 +77,48 @@ func (s *Service) Create(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	newUser.EmailToken = CheckEmail(newUser)
+	EmailVerificationToken(req.Email)
```

Comment:
the return value is ignored here, it should probably be saved to a db or something